src/components/DotNavigation/DotNavigation.tsx

```diff
@@ -311,10 +311,10 @@ export class DotNavigation extends React.Component<
 
   render = () => {
     const { length, ...rest } = this.props;
-    let short = length <= MAX_SHORT_LIST_LENGTH;
+    const short = length <= MAX_SHORT_LIST_LENGTH;
 
     return _isNumber(length) && length > 0 ? (
-      <div {...styles('root', {short}, rest)} {...this._getDataAttributes()}>
+      <div {...styles('root', { short }, rest)} {...this._getDataAttributes()}>
         {length <= MAX_SHORT_LIST_LENGTH
           ? this.renderShortVersion()
           : this.renderLongVersion()}
```

Comment:
Note that you have this lint correction in all your PR's
So i'll add this fix to master, and then remove these changes and pull rebase